### PR TITLE
Add PluralRules interface.

### PR DESCRIPTION
**Pull request by jpmedley**
Related to this: https://www.chromestatus.com/features/5653874773852160
javascript/builtins/Intl.json is already up to date.

### CODE REVIEW

File: javascript/builtins/PluralRules.json [hunk 1]
```diff
@@ -0,0 +1,208 @@
+{
+  "api": {
+    "OverconstrainedError": {
```
Comment (ddbeck) [inline javascript/builtins/PluralRules.json]: Copy and paste miss?

```suggestion
    "PluralRules": {
```

File: javascript/builtins/PluralRules.json [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          },
+          "webview_android": {
+            "version_added": null
```
Comment (ddbeck) [inline javascript/builtins/PluralRules.json]: The subfeatures have `"63"` for `webview_android`, should this be the same?

File: javascript/builtins/PluralRules.json [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "resolvedOptions": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PluralRules/resolvedOptions",
+          "support": {
+            "chrome": {
+              "version_added": null
```
Comment (ddbeck) [inline javascript/builtins/PluralRules.json]: For `chrome` and `chrome_android`, the subfeatures don't have data, but webview_android does. Is that intended?

**Comment by Elchi3** [comment]
Why has this been added when PlurarlRules is already in https://github.com/mdn/browser-compat-data/blob/master/javascript/builtins/Intl.json#L1270 ?

Also putting it in javascript/builtins/PluralRules.json but then hanging it off the api.PluarRules tree isn't something we should do.

I'm asking to remove this file in https://github.com/mdn/browser-compat-data/pull/3258.

**Comment by jpmedley** [comment]
This is just me not being as familiar with javascript/ as I am api/.

**Comment by ddbeck** [comment]
Compounded by me also not being very familiar with this. 😳 

See #3266 for the fix.

**Pull request merged**